backend/src/controller/plantings.rs

```diff
@@ -107,57 +94,51 @@ pub async fn create(
     params(
         ("map_id" = i32, Path, description = "The id of the map the layer is on"),
     ),
-    request_body = UpdatePlantingDto,
+    request_body = ActionDtoWrapperUpdatePlantings,
```

Comment:
Do we really want so long names? Why no simply UpdatePlantingsDto? (plural for collection)